python/py-pymysql/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-pymysql
```

Comment:
if you set `python.rootname PyMySQL` then you can remove the `master_sites` and `distname` fields and also get rid of `livecheck.type  pypi`